app/post.go

```diff
@@ -1205,12 +1205,17 @@ func (a *App) GetPostsForChannelAroundLastUnread(c request.CTX, channelID, userI
 	}
 	// Reset order to only include the last unread post: if the thread appears in the centre
 	// channel organically, those replies will be added below.
-	postList.Order = []string{lastUnreadPostId}
+	postList.Order = []string{}
```

Comment:
Do we need to initialize the slice here? I think a `nil` slice would work just as well (and avoids an allocation).

Reply:
I prefer collections to be empty rather than nil. Especially considering, this model will be JSON serialized. So, the empty collection would be easier to work with in JS.